modules/docs/llm/upgrade-guides/12.0-UPGRADE-GUIDE.md

```diff
@@ -784,7 +784,7 @@ things you'll want to keep in mind.
   dependencies on your own.
   - We recommend upgrading dependencies before running the codemod.
   - Always review your `package.json` files to make sure your dependency versions look correct.
-- The codemod will not handle every breaking change in v13. You will likely need to make some manual
+- The codemod will not handle every breaking change in this upgrade. You will likely need to make some manual
```

Comment:
This was spitting out `v13` into every upgrade guide – not really what we want. So I made it more generic.